js/modules/k6/http/http.go

```diff
@@ -37,36 +38,101 @@ import (
 	"github.com/dop251/goja"
 	"github.com/loadimpact/k6/js/common"
 	"github.com/loadimpact/k6/js/modules/k6/html"
+	"github.com/loadimpact/k6/lib"
 	"github.com/loadimpact/k6/lib/netext"
 	"github.com/loadimpact/k6/stats"
 	"github.com/pkg/errors"
+	"golang.org/x/crypto/ocsp"
 )
 
 var (
 	typeString = reflect.TypeOf("")
 	typeURLTag = reflect.TypeOf(URLTag{})
 )
 
+type OCSP struct {
+	ProducedAt, ThisUpdate, NextUpdate, RevokedAt int64
+	RevocationReason                              string
+	Status                                        string
+}
+
 type HTTPResponseTimings struct {
 	Duration, Blocked, LookingUp, Connecting, Sending, Waiting, Receiving float64
 }
 
 type HTTPResponse struct {
```

Comment:
This struct is honestly getting big enough that I'd split it off into a `response.go` file.